tasks/tempest_verifiers.yml

```diff
@@ -4,69 +4,75 @@
   debug:
     msg: "We are now configuring {{ item.key }}"
 
+- name: Select scenario {{ item.key }}
+  become_user: "{{ rally_user }}"
+  shell: "{{ rally_cmd }} deployment use {{ item.key }}"
+  changed_when: False
+  args:
+    executable: /bin/bash
+
 - name: Check existing rally verifiers
-  become_user: rally
-  shell: "source /home/rally/rally/bin/activate && rally deployment use {{ item.key }} &>/dev/null && rally verify list-verifiers | grep -v WARNING && deactivate"
+  become_user: "{{ rally_user }}"
+  shell: "source /home/rally/.rally/openrc && {{ rally_cmd }} verify list-verifiers | grep -v WARNING"
```

Comment:
Do we need the openrc for list verifiers? In  the original it sourced the virtualenv?

Reply:
Removed